### PR TITLE
feat:hello-world

### LINKED ISSUE #3: Hello World
Using a basic hello world program, this subsection under Getting Started shows the syntax of the main function, how to print something to the screen, and how to run the program (this assumes the user has followed the installation steps and has the cairo binaries registered in his $PATH) 

Estimated time: 1 hour

### PR DESCRIPTION
**Pull request by enitrat**
Closes #3

### CODE REVIEW

File: src/appendix-04-useful-development-tools.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Automatic Formatting with `cairo-format`
+
+The `cairo-format` tool reformats your code according to the community code style.
+Many collaborative projects use `cairo-format` to prevent arguments about which
+style to use when writing Cairo: everyone formats their code using the tool.
+
+To format any Cairo project, enter the following:
+
+```console
+cairo-format -r
```
Comment (fricoben) [inline src/appendix-04-useful-development-tools.md:16]: I didn't know that 😭

File: src/ch01-02-hello-world.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+```
+
+These lines define a function named `main`. The `main` function is special: it
+is always the first code that runs in every executable Cairo program. Here, the
+first line declares a function named `main` that has no parameters and returns
+nothing. If there were parameters, they would go inside the parentheses `()`.
+
+The function body is wrapped in `{}`. Cairo requires curly brackets around all
+function bodies. It’s good style to place the opening curly bracket on the same
+line as the function declaration, adding one space in between.
```
Comment (fricoben) [inline src/ch01-02-hello-world.md:96]: is this text really useful ? I mean I think with a very basic understanding of programmation you already understood that.

File: src/ch01-02-hello-world.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+> Note: If you want to stick to a standard style across Cairo projects, you can
+> use the automatic formatter tool called `cairo-format` to format your code in a
```
Comment (fricoben) [inline src/ch01-02-hello-world.md:99]: You already said it but I think that's a good point to have a notes like this 🔥

**Review by fricoben — COMMENTED** [review]
Just a question about the amount of explaination you gave

File: src/appendix-04-useful-development-tools.md (hunk 1 above)
Comment (enitrat) [reply src/appendix-04-useful-development-tools.md:16]: That's why we're writing docs my good sir 🤝

File: src/ch01-02-hello-world.md (hunk 2 above)
Comment (enitrat) [reply src/ch01-02-hello-world.md:96]: Idk tbh that's just how it's explained in the rust book and I followed the same template. I agree that this is not entirely necessary, but coming from Cairo >0.10 we used to have functions without curly braces :)

Comment (fricoben) [reply src/ch01-02-hello-world.md:96]: Better to explain too much than not enough

**Review by fricoben — APPROVED** [review]

**Pull request merged**